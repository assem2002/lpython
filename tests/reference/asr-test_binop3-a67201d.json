{
    "basename": "asr-test_binop3-a67201d",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_binop3.py",
    "infile_hash": "eb43132639edd47d89802b8693880ed350800a88c7c5f94eb93a9b3b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_binop3-a67201d.stderr",
    "stderr_hash": "0a9565ce5ba341309a0ea2a81b1eed455a4043ee6c824c1465fc3df1",
    "returncode": 2
}