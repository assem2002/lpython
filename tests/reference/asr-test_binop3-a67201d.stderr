semantic error: Type mismatch in binary operator; the types must be compatible
 --> tests/errors/test_binop3.py:5:9
  |
5 |     y = complex(5)+100
  |         ^^^^^^^^^^ ^^^ type mismatch (c32 and i32)


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
